GitHubActionsTestLogger/TestReporterContext.cs

```diff
@@ -27,32 +29,42 @@ private string FormatAnnotation(string format, TestResult testResult)
 
         // Name token
         buffer
-            .Replace("@test", testResult.TestCase.DisplayName)
+            .Replace("@test", testNode.DisplayName)
             // Backwards compat
-            .Replace("$test", testResult.TestCase.DisplayName);
+            .Replace("$test", testNode.DisplayName);
 
         // Trait tokens
-        foreach (var trait in testResult.Traits.Union(testResult.TestCase.Traits))
+        // TODO: this is not enough, we would also need to handle properties from VSTest bridge
```

Comment:
Is that a big piece of work, or relatively trivial?

Reply:
The VSTest bridge is deprecated with .NET 10, so this can be skipped?